### ISSUE TITLE
Remove user-scalable attribute from theme

### ISSUE THREAD

**Issue by josephfusco** [campuspress/divi-accessibility#4]
Viewport meta has `user-scalable=0` which is not a valid value. It can be assumed that the intended goal was to set `user-scalable=no`.

For a11y reasons, sites should not suppress or limit the ability of users to resize a document, and having user-scalable present creates false positives on accessibility audits.

This attribute should be removed altogether for accessibility auditing reasons and to eliminate any future issues if Divi does update eventually with a valid value.

**Comment by josephfusco** [comment]
Fixed via 237be53182b5ee340a6aa05e2a9b10b87b938cdc
